includes/templates/template_default/templates/tpl_index_default.php

```diff
@@ -32,7 +32,11 @@
   $show_display_category = $db->Execute(SQL_SHOW_PRODUCT_INFO_MAIN);
   while (!$show_display_category->EOF) {
 ?>
-
+<?php
+/**
+ * display the Featured Categories Center Box
```

Comment:
```suggestion
 * display the Featured Products Center Box
```

Reply:
main_page=featured_categories uses the display code from brands. I don't remember why but I'm sure it felt like a good idea at the time. Give me a few and I can replace it with the front page code.